engine-plugins/spin-plugin/pom.xml

```diff
@@ -133,6 +133,12 @@
       <artifactId>jaxb-impl</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.1</version>
+    </dependency>
```

Comment:
this is a bit unexpected. I would not expect any required `javax` dependency. I have to take a deeper look.